src/content/lodash/number/random/vanilla.js

```diff
@@ -0,0 +1,28 @@
+const random = (lower = 0, upper = 1, floating) => {
```

Comment:
During the #206 investigation, I've found the following discrepancies comparing to `lodash` implementation:
1. `_.random(-1)` returns integers in `[-1, 0]`, while this solution will return numbers in `[-1, 1]`
2. trivial solution (updating `upper` to equal `0` by default won't work), since `_.random()` should return integers in `[0, 1]`

Reply:
I'll add the test case (it might take a while as I'm swamped with work these days)